docs/sources/mimir/manage/tools/mimirtool.md

```diff
@@ -193,15 +222,31 @@ Within the output dir, the configuration file is named `config.yaml` and the tem
 When you use the `--output-dir` flag, the command only writes the output to files and doesn't print the configuration to the console.
 {{< /admonition >}}
 
-Once you have migrated your configuration, verify it using the verify command and the
-`--utf8-strict-mode` flag:
+Once you have migrated an Alertmanager configuration, verify it using the verify command:
+
+```bash
+mimirtool alertmanager verify <config_file> [template_files...]
+```
+
+It should succeed without warnings.
```

Comment:
I think we can probably remove this line- it's self-evident.

Reply:
I thought about this, but in this specific case I would like to make it as explicit as possible, as it's quite an important step for customers to run against their configurations.

Reply:
I thought about this, but in this specific case I would like to make it as explicit as possible, as it's quite an important step for customers to run against their configurations.